RadarSDK/RadarVerificationManager.m

```diff
@@ -328,4 +337,233 @@ - (void)getAttestationWithNonce:(NSString *)nonce completionHandler:(RadarVerifi
     }
 }
 
+// inspired by https://github.com/securing/IOSSecuritySuite
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wundeclared-selector"
+- (BOOL)isJailbroken {
+    BOOL jailbroken = NO;
+    
+    NSError *error = nil;
+    NSFileManager *fileManager = [NSFileManager defaultManager];
+    
+    // URL scheme check
+    NSArray *suspiciousURLSchemes = @[
+        @"undecimus://",
+        @"sileo://",
+        @"zbra://",
+        @"filza://"
+    ];
+    for (NSString *urlScheme in suspiciousURLSchemes) {
+        NSURL *url = [NSURL URLWithString:urlScheme];
+        if ([[UIApplication sharedApplication] canOpenURL:url]) {
+            [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelDebug message:@"Jailbreak check failed: URL scheme check"];
```

Comment:
worth adding the failed scheme to the log?

Reply:
Will do